lookout/style/format/features.py

```diff
@@ -83,6 +84,12 @@ def from_node(node: bblfsh.Node, file: str, path: str) -> Iterable["VirtualNode"
                               node=node, path=path)
             return
         start_offset = outer_token.find(node.token)
+        assert start_offset >= 0, (
+            "Couldn't find the token in the specified position:\nNode role: %s\nParsed form: “%s”"
```

Comment:
File information is needed

Reply:
file information is already present where all the asserts are caught